main.go

```diff
@@ -174,6 +195,7 @@ func GetStaticConfigs(svc kafkaClient, opt_filter ...regexp.Regexp) ([]Prometheu
 }
 
 func main() {
+	flag.Var(&tagFilters, "tag", "A key=value for filtering by tags. Flag can be specified multiple times, resulting OR expression.")
```

Comment:
Was there a reason for defining this flag in `main()` rather than with all of the other flags at the top of the file?

Reply:
flag.Var() is a void function, so I cannot use the group var statement. 
Thinking about it, `tagFilters` shouldn't be global either. Now that we process and pass a Filter struct, and the construction is in main.